packages/components/src/components/input-date/input-date.e2e.ts

```diff
@@ -36,6 +37,32 @@ test.describe('kol-input-date', () => {
 			}, TEST_DATE);
 			await expect(page.locator('input')).toHaveValue('04:02');
 		});
+		test('should return Date when initial value is a Date object and trigger onChange', async ({ page }) => {
+			await page.setContent('<kol-input-date _label="Date input"></kol-input-date>');
+			await page.locator('kol-input-date').evaluate((element: HTMLKolInputDateElement, date) => {
+				element._value = date;
+				element.addEventListener('change', (e: Event) => {
+					const value: Date | string = (e as CustomEvent).detail.value;
+					expect(value).toBeInstanceOf(Date);
```

Comment:
Diese Expectations werden nicht ausgeführt, vermutlich weil sie asynchron aufgerufen werden.

Expectations sollten immer synchron (oder nach einem `await`) auf der oberen Ebene und am Ende des `test`()-Blocks ausgeführt werden.

Ein Beispiel, wie das asynchron funktionieren kann, findest du hier: `packages/components/src/components/select/select.e2e.ts:16`

Tipp: Wenn du nicht sicher bist, ob eine Expectation korrekt funktioniert, versuche sie "kaputt" zu machen.
In diesem Beispiel könntest du den Aufruf z.B. ändern zu `expect(value).toBeInstanceOf(Map);`. Die Tests sollten dann fehlschlagen. Wenn sie das nicht tun, stimmt etwas mit dem Test nicht.